docs/News.md

```diff
@@ -1,3 +1,10 @@
+## June 12, 2019
+ *   **Added data**  of 1350 samples from 3 studies:
```

Comment:
The wording "Added data of" doesn't sound right.  In the future can you say "Added 1350 samples.." or "Added data consisting of....".

Reply:
Sure, will modify this statement for the future releases.